spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDefinitionController.java

```diff
@@ -16,10 +16,19 @@
 
 package org.springframework.cloud.dataflow.server.controller;
 
+import java.util.ArrayList;
```

Comment:
Yeah... We really should decide something common for all our projects.
Because we use in SI, SA, SK etc. these import options:
````xml
<module name="ImportOrder">
	<property name="groups" value="java,/^javax?\./,org,org.springframework,*" />
	<property name="ordered" value="true" />
	<property name="separated" value="true" />
	<property name="option" value="top" />
	<property name="sortStaticImportsAlphabetically" value="true" />
</module>
````
:cry: